src/Xamarin.Android.Build.Tasks/Tasks/Lint.cs

```diff
@@ -185,7 +185,9 @@ public override bool Execute ()
 				return false;
 			}
 
+			Log.LogDebugMessage ("  LintTool: {0}", GenerateFullPathToTool());
```

Comment:
This is a `ToolTask` subclass; the full path to the utility used *plus* all command-line arguments should already be in the diagnostic build output. This line should be redundant.